main.cpp

```diff
@@ -52,84 +52,114 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: Employee
+//  action 1: Performs Task
+employee.performTask();
 
-//  action 2:
+//  action 2: Clocks in
+employee.clockIn();
 
-//  action 3:
+//  action 3: Rotates Employee
+employee.rotate();
 
 //  2)
-//  Noun:
-//  action 1:
+//  Noun: Student
+//  action 1: Leaves home and heads to school
+student.goToSchool();
 
-//  action 2:
+//  action 2: Student goes from homework to class
+student.goToClass();
 
-//  action 3:
+//  action 3: Student does homework
+student.doHomework();
 
-//  3)
-//  Noun:
-//  action 1:
+//  3) 
+//  Noun: Bus Driver
+//  action 1: Starts the bus
+busDriver.startBus();
 
-//  action 2:
+//  action 2:Stops at bus stop
+busDriver.makeStop();
 
-//  action 3:
+//  action 3: heads to station
+busDriver.goToStation();
 
-//  4)
-//  Noun:
-//  action 1:
+//  4) 
+//  Noun: Doctor
+//  action 1: Performs checkup
+doctor.performCheckup();
 
-//  action 2:
+//  action 2: Orders bloodwork
+doctor.orderBloodwork();
 
-//  action 3:
+//  action 3: Performs surgery
+doctor.performSurgery();
 
 //  5)
-//  Noun:
-//  action 1:
+//  Noun: Elevator
+//  action 1: moves to specified floor
+elevator.gotoNthFloor();
 
-//  action 2:
+//  action 2: Stops at floor
+elevator.stop();
 
-//  action 3:
+//  action 3: opens door
+elevator.open();
 
 //  6)
-//  Noun:
-//  action 1:
+//  Noun: Fireman
+//  action 1: puts on fire equipment
+fireman.getSuitedUp();
 
-//  action 2:
+//  action 2: slides down firehouse pole
+fireman.slideDownPole();
 
-//  action 3:
+//  action 3: outs out fire
+fireman.putOutFire();
 
 //  7)
-//  Noun:
-//  action 1:
+//  Noun: ConvenienceClerk
+//  action 1: Stocks shelves
+convenienceClerk.stockShelves();
 
-//  action 2:
+//  action 2: rings out customer
+convenienceClerk.ringUpCustomer();
 
-//  action 3:
+//  action 3: takes down thief
+convenienceClerk.takeDownThief();
 
 //  8)
-//  Noun:
-//  action 1:
+//  Noun: Thief
+//  action 1: puts on mask
+thief.putOnMask();
 
-//  action 2:
+//  action 2: calls get away driver
+thief.callDriver();
 
-//  action 3:
+//  action 3: robs convenience store
+thief.robConvenienceStore();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.